.build/run-ci.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# variables, with defaults
+[ "x${CASSANDRA_DIR}" != "x" ] || CASSANDRA_DIR="$(readlink -f $(dirname "$0")/..)"
+[ "x${KUBECONFIG}" != "x" ]    || KUBECONFIG="${HOME}/.kube/config"
+[ "x${KUBE_NS}" != "x" ]       || KUBE_NS="default" # FIXME – doesn't work in other namespaces :shrug:
+
+# pre-conditions
+command -v kubectl >/dev/null 2>&1 || { echo >&2 "kubectl needs to be installed"; exit 1; }
+command -v helm >/dev/null 2>&1 || { echo >&2 "helm needs to be installed"; exit 1; }
+
+# Parse command-line arguments
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        -c|--kubeconfig)
+            KUBECONFIG="$2"  # This sets the KUBECONFIG variable to the next argument.
+            shift            # This shifts the arguments to the left, discarding the current argument and moving to the next one.
+            shift            # This is an additional shift to move to the argument after the option value.
+            ;;
+        -ctx|--kubecontext)
+            unset KUBECONFIG
+            KUBECONTEXT="$2" # This sets the KUBECONTEXT variable to the next argument.
+            shift            # This shifts the arguments to the left, discarding the current argument and moving to the next one.
+            shift            # This is an additional shift to move to the argument after the option value.
+            ;;
+        *)
+            echo "Unknown option: $1"
```

Comment:
```suggestion
            echo "Unknown option: $1"
        -h|--help)
            echo "todo – write up help msg"
```

Reply:
i'm also keen to see a parameterisation framework here, as there are a number of parameters to add.

- git url and branch
- whether to tear down the jenkins cluster
- include and exclude patterns on stage names
- any other arbitrary variables to pass into jenkins